packages/core/core/src/Parcel.js

```diff
@@ -131,7 +129,7 @@ export default class Parcel {
       // console.log('Finished build'); // eslint-disable-line no-console
       return bundleGraph;
     } catch (e) {
-      if (e !== abortError) {
+      if (e.code !== 'BUILD_ABORTED') {
```

Comment:
I'd rather import the class and do an `instanceof` check here since it's more type-checker friendly, but we've been using `code` elsewhere. Anyone have a preference?

Reply:
Yeah, seems like using an `instanceof` check would be less likely to break things if the code is changed. @devongovett, what was the reasoning for checking against e.code in other places.

Reply:
Just grepped through things and it looks like we only reference `err.code` to switch on Node's own error objects, which use `code` to differentiate themselves. I think it makes sense that our own errors subclass Error instead.